.github/workflows/test.yml

```diff
@@ -27,100 +27,120 @@ jobs:
   api-breakage:
     if: ${{ !(github.event.pull_request.draft || false) }}
     runs-on: ubuntu-latest
-    container: swift:5.8-jammy
+    container: swift:jammy
```

Comment:
Why this change? It's kind of inconsistent with all other CIs.

Reply:
It's in line with https://github.com/vapor/ci/blob/main/.github/workflows/run-unit-tests.yml#L56 and https://github.com/vapor/ci/blob/main/.github/workflows/run-unit-tests.yml#L70 - these jobs don't care overmuch about which Swift version they use in particular, it should always just be the "latest stable" version, which is exactly what the non-versioned Docker tags provide.